app/javascript/components/admin/UserModal.js

```diff
@@ -85,7 +86,7 @@ class UserModal extends React.Component {
                 <Form.Control
                   id="email-input"
                   name="email"
-                  value={this.props.initialUserData.email ? this.props.initialUserData.email : ''}
+                  value={this.state.email ? this.state.email : ''}
```

Comment:
you can just do `this.state.email || ''` here for shorthand

Reply:
Thanks! [ea9253d6956658ca2db9763a9140c370e806cb31](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/ea9253d6956658ca2db9763a9140c370e806cb31)